caf-audit-service-container/README.md

```diff
@@ -20,7 +20,30 @@ The number of replica shards (copies) that each primary shard should have. e.g.
 
 ##### CAF\_AUDIT\_SERVICE\_API\_CONFIG\_PATH
 
-The path to the directory containing the config.properties which can be used as an alternative means of specifying the environment variables. e.g. .\config-props 
```

Comment:
This section should begin with CAF_LOG_LEVEL with a paragraph or set of paragraphs directly under (just like the other settings).

Does audit-service-deploy README not require a similar change?